igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -38,16 +38,18 @@ class JenkinsCache {
     @Value('${spinnaker.jedis.prefix:igor}')
     String prefix
 
+    String id = 'builds'
+
     List<String> getJobNames(String master) {
         Jedis resource = jedisPool.resource
-        List<String> jobs = resource.keys("${prefix}:${master}:*").collect { extractJobName(it) }.sort()
+        List<String> jobs = resource.keys("${prefix}:${id}:${master}:*").collect { extractJobName(it) }.sort()
```

Comment:
What are the implications for things that have already been indexed by Jenkins? This should not modify any existing jenkins keys as this can have catastrophic consequences on existing services already using jenkins and igor.